core/includes/theme.inc

```diff
@@ -2476,7 +2476,7 @@ function theme_head_tag($variables) {
  *   - title: A descriptive verb for the link, like 'Read more'.
  */
 function theme_more_link($variables) {
-  return '<div class="more-link">' . l(t('More'), $variables['url'], array('attributes' => array('title' => $variables['title']))) . '</div>';
+  return l(t('More'), $variables['url'], array('attributes' => array('title' => $variables['title'], 'class' => 'more-link')));
```

Comment:
After a second turn, I definitely think that this is an unrelated changes and should be reverted for now.